test/reducers/entities/posts.test.js

```diff
@@ -67,6 +93,36 @@ describe('Reducers.posts', () => {
                 },
             });
         });
+
+        it('should add postId to postsInChannel', () => {
```

Comment:
Similarly, we should clarify that this is only when `postsInChannel[channelId]` is set

Reply:
👍